packages/simulator/lib/vroom.js

```diff
@@ -40,13 +40,25 @@ module.exports = {
       },
     }
   },
-  taxiToVehicle({ id, position, passengerCapacity, heading, passengers }, i) {
+  taxiToVehicle({ id, position, heading, plan }, i) {
     return {
       id: i,
       description: id,
-      capacity: [passengerCapacity - (passengers?.length || 0)],
       start: [position.lon, position.lat],
       end: heading ? [heading.lon, heading.lat] : undefined,
+      steps: plan.map((planStep) => {
+        console.log('step', planStep)
+        return {
+          type: planStep.action,
+          arrival: planStep.arrival,
+          duration: 100, // TODO: Calculate.
```

Comment:
@irony nu har jag lagt in steps här. Men jag funderar på _duration_ som är "cumulated travel time upon arrival at this step". Avser det den faktiska tiden som har gått, eller den teoretiska tiden som fordonet bör behöva i varje steg?